CONTRIBUTING.md

```diff
@@ -69,3 +69,13 @@ flake8 .
 
 You can set the `PYGFX_LOG_LEVEL` environment variable to get more
 detailed log messages. Can be an int or any of the standard level names.
+
+
+## Attributes
+
+We distinguish four kinds of attributes:
+
+* Public attibutes for the user: should in most cases be ``@property``'s.
+* Private attributes: prefixed with "_" as usual.
+* Attributes used by other parts of pygfx but not intended for the user: prefixed with "_gfx_".
```

Comment:
I don't understand how this is different from other private attributes.

Reply:
Well normally private attributes are only used by the object itself. There are a few cases where an attribute or method of one object is used by another in a different module or even subpackage. The idea here is to prefix such cases, which is a way of saying, yes I use a private var of another object, but this is intentional. Or in other words, it's private to pygfx :)